src/main/java/adris/altoclef/tasks/ResourceTask.java

```diff
@@ -85,6 +85,16 @@ protected void onStart(AltoClef mod) {
 
     @Override
     protected Task onTick(AltoClef mod) {
```

Comment:
This change needs reverted. I am not sure if the placement of this matters, but it should be at the end.